enhancements/console/dynamic-plugins.md

```diff
@@ -433,6 +433,33 @@ const VMHeading = () => {
 };
 ```
 
+For a 4.11 release a `console.openshift.io/default-i18next-namespace` annotation,
```

Comment:
I'd make it clear that this annotation should only be used by plugins that need to work with 4.10 and 4.11. We'll eventually drop this annotation when 4.10 goes out of support in favor of the proper API.